### ISSUE TITLE
TensorRT 10.5 Flux Dit BF16 precision

### ISSUE THREAD

**Issue by QZH-eng** [NVIDIA/TensorRT#4215]
## Description

When I used TensorRT 10.5 to infer Flux Dit on A800 using BF16 dataType, I found that there was a significant decrease in accuracy, while there was no significant decrease in accuracy when I used Pytorch BF16 to infer

## Environment

**TensorRT Version**:

**NVIDIA GPU**: A800

**NVIDIA Driver Version**: 535.54.03

**CUDA Version**:12.2

**CUDNN Version**:

Operating System:

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by lix19937** [comment]
How about half or fp32 ?

**Comment by QZH-eng** [comment]
> How about half or fp32 ?

From the results, the precision of FP32 is relatively good, and FP16 will overflow

**Comment by lix19937** [comment]
Can you upload and zip the fp16 build log here ?

**Comment by yuanyao-nv** [comment]
cc @asfiyab-nvidia

**Comment by asfiyab-nvidia** [comment]
@QZH-eng Is this for Flux.1-dev or Flux.1-schenll? Can you run the below commands and share the images please? The images generated should be almost identical for the TRT and PyTorch pipelines with the same seed and no warmup

```
# PyTorch
python3 demo_txt2img_flux.py "a beautiful photograph of Mt. Fuji during cherry blossom" --hf-token=$HF_TOKEN --bf16 --num-warmup-runs 0 --seed 46 --torch-inference 0

# TensorRT
python3 demo_txt2img_flux.py "a beautiful photograph of Mt. Fuji during cherry blossom" --hf-token=$HF_TOKEN --bf16 --num-warmup-runs 0 --seed 46
```
